instance/models/mixins/openedx_config.py

```diff
@@ -486,6 +486,10 @@ def _get_configuration_variables(self):
             template["forum_source_repo"] = "https://github.com/open-craft/cs_comments_service.git"
             template["forum_version"] = "opencraft-release/juniper.3"
             template["FORUM_VERSION"] = "opencraft-release/juniper.3"
+        if self._is_openedx_release_in(['koa']):
```

Comment:
@0x29a, can you also add some details about the code drift/commits that require us to use the fork?

Reply:
@0x29a, also since we have no instances on Ironwood and it is not supported anyways, can we remove the if condition for it and the other ones above?

This function is becoming becoming messy and we will have to clean it up properly if we want to have release-specific configuration for multiple unsupported releases.

Reply:
@lgp171188, we have periodic builder for Ironwood: https://manage.opencraft.com/instance/19890/.

Reply:
@0x29a, wouldn't it be simpler and easier to just add the configuration override to that one instance?

Reply:
You right, @lgp171188. By the way, we have `hawthorn.1` instance: https://manage.opencraft.com/instance/13740/

Refactoring this would require testing the deployment for both instances. Would it make sense to create a follow-up ticket instead of doing this withing this PR?

Reply:
@0x29a, the issue with follow-up tasks is that they will get pushed out and deprioritized for various reasons and might get lost. And then we will end up with ugly code that is probably not needed any more. Can we just add the configuration overrides to the affected instances and add a note not to redeploy them and proceed with the upgrade? Checking and fixing issues, if any, on those tasks can be a follow-up task.

Reply:
Makes sense, @lgp171188. I will do this shortly.

Reply:
Done, @lgp171188. Here is an extra configuration I added for the Hawthorn instance:
```yaml
# master and juniper release onwards has djangoapps.heartbeat installed by default
# openedx <= ironwood requires this if celery check is included in
# heartbeat extended checks (which we add by default above)
EDXAPP_CMS_ENV_EXTRA:
  ADDL_INSTALLED_APPS: [ 'openedx.core.djangoapps.heartbeat' ]
EDXAPP_LMS_ENV_EXTRA:
  ADDL_INSTALLED_APPS: [ 'openedx.core.djangoapps.heartbeat' ]
  HEARTBEAT_EXTENDED_CHECKS:
    - lms.lib.comment_client.utils.check_forum_heartbeat
```

For Ironwood:
```yaml
# master and juniper release onwards has djangoapps.heartbeat installed by default
# openedx <= ironwood requires this if celery check is included in
# heartbeat extended checks (which we add by default above)
EDXAPP_CMS_ENV_EXTRA:
  ADDL_INSTALLED_APPS: [ 'openedx.core.djangoapps.heartbeat' ]
EDXAPP_LMS_ENV_EXTRA:
  ADDL_INSTALLED_APPS: [ 'openedx.core.djangoapps.heartbeat' ]
  HEARTBEAT_EXTENDED_CHECKS:
    - lms.lib.comment_client.utils.check_forum_heartbeat

# See https://github.com/edx/cs_comments_service/pull/323 and https://github.com/open-craft/cs_comments_service/pull/5
forum_source_repo: https://github.com/open-craft/cs_comments_service.git
forum_version: opencraft-release/ironwood.2
```